spec/theme/plugins.spec.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+describe(\Dxw\MyTheme\Theme\Plugins::class, function () {
+    beforeEach(function () {
+        \WP_Mock::setUp();
+        \WP_Mock::wpFunction('esc_html', [
+            'return' => function ($a) {
+                return '_'.$a.'_';
```

Comment:
Depends: we're mocking `admin_url` to return `__http://localhost/wp-admin/plugins.php__` -if that's for a different reason then I'd say this is a bit confusing. 

On the other hand, this is just a generally fiddly situation to test: we want to test both that the escaping function is called, and that the output of that function is ultimately displayed. The 'underscores' method elegantly achieves both of those in one step but is a bit obscure and leaves us with a test that is unrealistic.

The normal solution to this is to have two small unit tests: one which tests that the function is called (using a mock), and the other that tests that the result is displayed (using a stub), but since these tests take a lot of setup, that's maybe not ideal (?).

Perhaps this is highlighting that this class has too many responsibilities: that the code which adds a notice belongs in another object, and the tests for the `Plugins` class would only test that this object is called.

That's a bit of work though, so it's up to you to decide whether it's worth it: I'm happy to approve the PR as-is.